numpy/f2py/tests/test_semicolon_split.py

```diff
@@ -23,6 +26,10 @@ class TestMultiline(util.F2PyTest):
 end python module {module}
     """.format(module=module_name)
 
+    @pytest.mark.skipif(platform.system() == 'Darwin',
+                        reason="Prone to error when run with "
+                               "numpy/f2py/tests on mac os, "
+                               "but not when run in isolation")
```

Comment:
What does this mean?

Reply:
It means that running `python runtests.py -t "numpy/f2py/tests/test_semicolon_split.py"` does not cause issues, but the decorated test will error with `python runtests.py -t "numpy/f2py/tests"`, or the entire test suite. So, context-dependent behavior in the test container.